linux_os/guide/system/software/integrity/crypto/openssl_use_strong_entropy/ansible/shared.yml

```diff
@@ -0,0 +1,12 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: "copy a file with shell snippet to configure openssl strong entropy"
```

Comment:
`copy` is the Ansible module in use, but we are not copying content but dumping / putting.
Suggestion: `Put (or setup) a shell snippet to configure OpenSSL to always use strong entropy`.

Also, mind the OpenSSL capital letters. :)